Makefile

```diff
@@ -366,7 +366,8 @@ _test.integration: _check.container.environment go-junit-report
 
 .PHONY: test.integration.dbless.knative
 test.integration.dbless.knative:
-	@$(MAKE) _test.integration \
+	@KONG_CONTROLLER_FEATURE_GATES="Knative=true" \
```

Comment:
It's already set below, isn't it? 🤔 

Patryk's https://github.com/Kong/kubernetes-ingress-controller/pull/3975 already solved the issue of failing Knative tests.

Reply:
Close this.